### ISSUE TITLE
update screenshots for 15

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/nextcloud.com#1020]
We need to update screenshots for 15. Ideally, all get take again, but that is hard in some cases, requiring a call etc.

What needs to happen is:
* create new screenshot, 16:9 (!!!) of any resolution (1600x900, 1270x720, 1920x1080, etc) and put in features folder
* update html of the page:

1. make sure there's a ```<div class="row">``` with in it a ```<div class="col-md-6">```
2. Add to the ```col-md-6``` the following classes: ```image--feature``` (might be there already) and ```new-img```
3. update image URL! make sure image has the class ```featureimg```
4. Test! If it doesn't look good, double-check there's a row and it is ```col-md-6```, others are not (yet) supported.

Pages to do:
- [ ] https://nextcloud.com/files/
- [ ] https://nextcloud.com/groupware/
- [ ] https://nextcloud.com/talk/
- [ ] https://nextcloud.com/athome/
- [ ] https://nextcloud.com/secure/
- [ ] https://nextcloud.com/globalscale/
- [ ] https://nextcloud.com/file-drop/
- [ ] https://nextcloud.com/sharing/
- [ ] https://nextcloud.com/collaboraonline/
- [ ] https://nextcloud.com/onlyoffice/
- [ ] https://nextcloud.com/outlook/
- [ ] https://nextcloud.com/workflow/
- [ ] https://nextcloud.com/clients/
- [ ] https://nextcloud.com/storage/
- [ ] https://nextcloud.com/monitoring
- [ ] https://nextcloud.com/about/
- [ ] https://nextcloud.com/healthcare/
- [ ] https://nextcloud.com/education/
- [ ] https://nextcloud.com/release-channels/
- [ ] https://nextcloud.com/connect/
- [ ] https://nextcloud.com/federation/
- [ ] https://nextcloud.com/securesharing/

**Comment by jancborchardt** [comment]
@MariusBluem @jospoortvliet @nextcloud/website for any new screenshots we need to make sure:

- They are sufficiently zoomed in so the elements are nicely visible
- There’s no light blue background around it

This especially makes it easier to view them on mobile (narrow screens), but also for everyone generally.


Compare this (currently on https://nextcloud.com/secure/):
![](https://nextcloud.com/wp-content/themes/next/assets/img/features/sharing_admin-nw.png?x16328)

And this screenshot I just made on full HD with 180% zoom in Firefox (yes, Ubuntu doesn’t handle High-DPI properly, but the point is that everything is visible and there’s no unnecessary whitespace on the right):
![nextcloud 180](https://user-images.githubusercontent.com/925062/49578455-1630e780-f94a-11e8-9b48-8e91ac002bc1.png)

**Comment by jospoortvliet** [comment]
@jancborchardt the new screenshots don't have the blue anymore, I add the 'browser chrome' and the background with css. We can change the look later, I don't want to break the design because it does look nice.

Zoom in the screenshots is fine - as much as wanted, it doesn't matter much ;-)

As long as the screenshot is 16:9.

**Comment by MariusBluem** [comment]
I will push updated screenshots tonight, with 16:9, zommed in and all the other requirements. 📸📸

**Comment by MariusBluem** [comment]
Please review if this would be OK: 
https://cloud.nextcloud.com/s/w2PzwkqKzsi4F2F

More than 170% seems to be a bit messy (layout-wise)

**Comment by jospoortvliet** [comment]
@MariusBluem yes, that all looks good! Run them through https://tinypng.com & put them in a PR with the html changes (if you can, if not, I can help).

Great work!

**Comment by jancborchardt** [comment]
These look great @MariusBluem, thank you so much! Finally nicely sized screenshots with visible content! :wink:
